### PR TITLE
galliumnine02: new verb

### PR DESCRIPTION
**Pull request by dhewg**
See https://github.com/iXit/wine-nine-standalone
This adds the current release https://github.com/iXit/wine-nine-standalone/releases/tag/v0.2

### CODE REVIEW

File: src/winetricks [hunk 1]
```diff
@@ -8670,6 +8670,33 @@ load_flash()
...
+    year="2019" \
+    media="download" \
+    file1="gallium-nine-standalone-v0.2.tar.gz" \
+    installed_file1="$W_SYSTEM32_DLLS_WIN/d3d9-nine.dll" \
+    installed_file2="$W_SYSTEM32_DLLS_WIN/ninewinecfg.exe" \
+    homepage="https://github.com/iXit/wine-nine-standalone"
+
+load_galliumnine02()
+{
+    w_download "https://github.com/iXit/wine-nine-standalone/releases/download/v0.2/$file1"
```
Comment (austin987) [inline src/winetricks]: $file1 can't be used with w_download (it breaks the tests), please use the explicit filename.

Comment (dhewg) [reply src/winetricks]: Ok, done

Comment (austin987) [reply src/winetricks]: Thanks

**Comment by dhewg** [comment]
Thanks!

**Pull request merged**